src/Module/Followers.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * @file src/Module/Followers.php
+ */
+namespace Friendica\Module;
+
+use Friendica\BaseModule;
+use Friendica\Protocol\ActivityPub;
+use Friendica\Core\System;
+use Friendica\Model\User;
+
+/**
+ * ActivityPub Followers
+ */
+class Followers extends BaseModule
+{
+	public static function init()
```

Comment:
I'm querying ```$a->argv[1]``` some lines later, so I think we have to keep it.

Reply:
P.S.: Why using "content" at all? These are technical endpoints that never will be embedded in our page layout at all. So we should assure that they are called as early as possible in the whole process.

Reply:
The rationale is
- Module output is in `content()`. This is where we send the content back to the browser.
- Module processing is on `post()`. This is where we use the POST parameters to effect changes in the database.
- Module initialization common to both is in `init()`. But this shouldn't either display something nor effect change in the database.

For the errors I'm sorry I forgot I dropped the `Friendica\App` parameter for the module methods, `self::getApp()` is perfectly valid.

I'd still like us to use `content()` for outputting content instead of `init()`.

Reply:
I think that ```content(...)``` is for page content, not for protocol content. Just have a look at the complex content handling here: https://github.com/friendica/friendica/blob/develop/index.php#L380-L390

I do see your point why "init" is not ideal. But I would vote more for some additional class function that is reserved for protocol endpoints than using "content".